src/solver/conv_multipass_wino3x3WrW.cpp

```diff
@@ -381,15 +381,33 @@ bool ConvWinograd3x3MultipassWrW<WinoDataH, WinoFilterH, WinoDataW, WinoFilterW>
         if(miopen::IsDisabled(MIOPEN_DEBUG_AMD_WINOGRAD_MPASS_F3X3{}) ||
            params.kernel_stride_h == 1)
             return false;
-    if(wino_data_tile == 3 && wino_filter_tile == 4)
-        if(miopen::IsDisabled(MIOPEN_DEBUG_AMD_WINOGRAD_MPASS_F3X4{}))
-            return false;
-    if(wino_data_tile == 3 && wino_filter_tile == 5)
-        if(miopen::IsDisabled(MIOPEN_DEBUG_AMD_WINOGRAD_MPASS_F3X5{}))
-            return false;
-    if(wino_data_tile == 3 && wino_filter_tile == 6)
-        if(miopen::IsDisabled(MIOPEN_DEBUG_AMD_WINOGRAD_MPASS_F3X6{}))
-            return false;
+
+    const std::string name = params.GetStream().GetDeviceName();
+    if(params.IsFp16() && StartsWith(name, "gfx908") && params.kernel_stride_h == 1)
```

Comment:
[Recommendation]
```suggestion
    if(params.IsFp16() && StartsWith(name, "gfx908"))
```
@daniellowell what is your opinion?

Reply:
SWDEV_234193 does not contain conv3x3 stride2, only stride 1

Reply:
Ok, but are these solvers safe for stride 2?

Reply:
It can pass verification steps. @Kirpich30000 ?

Reply:
Stride 1 solvers are passing verification as well, however we see the problems. Perhaps the SWDEV is not related to these solvers at all. Well, it _is related_, but _the reason_ could be different. For example, by disabling these solvers we are freeing some workspace, thus allowing some other workspace-greedy solver to run _instead_ of some other solver, which needs less workspace but is really faulty.

We need complete logs collected at level 5 for analysis.

As far as we do not have information about the real cause of the issue, let's act as simple as possible, but with a guaranteed effect.

Reply:
better safe than sorry. ok

Reply:
And we have several log files in the ticket.